packages/client/src/object/fetchPage.ts

```diff
@@ -205,12 +243,46 @@ export async function fetchPageInternal<
   args: FetchPageArgs<Q, L, R, A> = {},
 ): FetchPageResult<Q, L, R> {
   if (objectType.type === "interface") {
-    return await fetchInterfacePage(client, objectType, args, objectSet) as any; // fixme
+    return await fetchInterfacePage(
```

Comment:
Not sure why this got reformatted, but no change here otherwise

Reply:
dprint will preserve newline whitespace often if it exists. lets set this back to the way it was before merging.